app/models/registration-schema.ts

```diff
@@ -50,7 +50,9 @@ export interface Answer {
 }
 
 export interface Answers {
```

Comment:
I've been using these guys and they seem pretty solid thus far:

```typescript
export interface Answer<T> {
    comments: any[]; // String?
    extra: object[];
    value?: T;
}

export interface RegistrationMetadata {
    [qid: string]: Answer<string | RegistrationMetadata>;
}
```

Reply:
WFM